docs/fundamentals/flow/executor-args.md

```diff
@@ -12,7 +12,7 @@
 | `uses_metas` | Dictionary of keyword arguments that will override the `metas` configuration in `uses` | `object` | `None` |
 | `uses_requests` | Dictionary of keyword arguments that will override the `requests` configuration in `uses` | `object` | `None` |
 | `py_modules` | The customized python modules need to be imported before loading the executor<br><br>Note that the recommended way is to only import a single module - a simple python file, if your<br>executor can be defined in a single file, or an ``__init__.py`` file if you have multiple files,<br>which should be structured as a python package. For more details, please see the<br>`Executor cookbook <https://docs.jina.ai/fundamentals/executor/executor-files/>`__ | `array` | `None` |
-| `port` | The port for input data to bind to, default is a random port between [49152, 65535] | `number` | `random in [49152, 65535]` |
+| `port` | The port for input data to bind to, default is a random port between [49152, 65535] | `string` | `random in [49152, 65535]` |
```

Comment:
are we sure? isn't this quite breaking?

Reply:
You can still pass a string and it will just get converted, so no breaking change from the user perspective.
This is necessary to be able to pass multiple ports for multiple replicas, and this follows the same syntax that was introduced by monitoring (where you pass multiple monitoring ports).